app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -231,10 +231,10 @@ const SendAdaPage = ({
     [updateAmount]
   )
 
-  const displayDropdownSelectedItem = ({assetName, policyId}: DropdownAssetItem) => (
+  const displayDropdownSelectedItem = ({assetName, policyId, type}: DropdownAssetItem) => (
     <div className="wrapper">
       {assetName}
-      {selectedAsset.type === AssetFamily.TOKEN && (
+      {type === AssetFamily.TOKEN && (
```

Comment:
is this related to the PR or just an improvement aside? I'd cosider naming the parameter `assetFamily`, type is too generic to give meaningful information to the reader

Reply:
It's not related to the PR, just improvement aside. It was using variable outside of function scope before. `type` is already existing member, it's not something introduced in this change.